crates/shared/src/http_solver.rs

```diff
@@ -173,6 +177,38 @@ impl HttpSolverApi for DefaultHttpSolverApi {
         serde_json::from_str(text)
             .with_context(|| format!("failed to decode response json, {}", context()))
     }
+
+    fn notify_auction_result(&self, auction_id: AuctionId, result: model::AuctionResult) {
+        let mut url = match self.base.join("notify") {
```

Comment:
this means the path for the call will be `/notify` instead of `notify_auction_result`?